src/modules/UI/scenes/WalletList/components/WalletListRow/FullWalletListRow.ui.js

```diff
@@ -148,19 +167,25 @@ class FullWalletListRowConnect extends Component<Props> {
                   </View>
                 )}
               </View>
-
-              <View style={[styles.rowBalanceTextWrap]}>
-                <View style={{ flexDirection: 'row', justifyContent: 'flex-end' }}>
-                  <T style={[styles.rowBalanceAmountText]}>{finalCryptoAmount}</T>
+              {this.props.isWalletFiatBalanceVisible ? (
+                <View style={[styles.rowBalanceTextWrap]}>
+                  <View style={{ flexDirection: 'row', justifyContent: 'flex-end' }}>
```

Comment:
I know this is a quick adjustment of the style, but we shouldn't have inline styles.  This should be in the style sheet.